internal/core/items/flickable.rs

```diff
@@ -318,25 +318,29 @@ impl FlickableData {
     fn mouse_released(inner: &mut FlickableDataInner, flick: Pin<&Flickable>, event: MouseEvent) {
         if let (Some(pressed_time), Some(pos)) = (inner.pressed_time, event.position()) {
             let dist = (pos - inner.pressed_pos).cast::<f32>();
-            let speed =
-                dist / ((crate::animations::current_tick() - pressed_time).as_millis() as f32);
 
-            let duration = 250;
-            let final_pos = ensure_in_bound(
-                flick,
-                (inner.pressed_viewport_pos.cast() + dist + speed * (duration as f32)).cast(),
-            );
-            let anim = PropertyAnimation {
-                duration,
-                easing: EasingCurve::CubicBezier([0.0, 0.0, 0.58, 1.0]),
-                ..PropertyAnimation::default()
-            };
-            (Flickable::FIELD_OFFSETS.viewport + Rectangle::FIELD_OFFSETS.x)
-                .apply_pin(flick)
-                .set_animated_value(final_pos.x, anim.clone());
-            (Flickable::FIELD_OFFSETS.viewport + Rectangle::FIELD_OFFSETS.y)
-                .apply_pin(flick)
-                .set_animated_value(final_pos.y, anim);
+            let millis = (crate::animations::current_tick() - pressed_time).as_millis();
+            if dist.square_length() > (DISTANCE_THRESHOLD * DISTANCE_THRESHOLD) as f32 && millis > 1
```

Comment:
Perhaps this is also worth mentioning in the changelog?

Reply:
This particular line is just not doing an animation if the mouse hasn't moved